packages/core/src/application.ts

```diff
@@ -470,6 +550,10 @@ export type ShutdownOptions = {
  * Configuration for application
  */
 export interface ApplicationConfig {
+  /**
+   * Name of the application context
+   */
+  name?: string;
```

Comment:
Is this really needed? Can we use the application class name so that no configuration is necessary?

Reply:
It's an optional property. By default, we use the application class name.